src/matrixeventprocessor.ts

```diff
@@ -205,6 +210,11 @@ export class MatrixEventProcessor {
             msg += `set the topic to \`${event.content!.topic}\``;
         } else if (event.type === "m.room.member") {
             const membership = event.content!.membership;
+            if (membership === "join") {
+                this.mxUserProfileCache.delete(`${event.room_id}:${event.sender}`);
```

Comment:
Alternatively, we could also set the new `membership` right away? (not 100% sure if it would need slight re-formatting)

Reply:
Huh? Not really, this is only used for setting webhook displaynames iirc so probably not much we can do?

Reply:
eeer, soru means `event.content`, sorry!

Reply:
OH

Reply:
i am dumb